README.md

```diff
@@ -1,25 +1,32 @@
 # NotificationCenter
+
+[![Build](https://github.com/DynamoDS/NotificationCenter/actions/workflows/build.yml/badge.svg)](https://github.com/DynamoDS/NotificationCenter/actions/workflows/build.yml)
+
+[![Publish](https://github.com/DynamoDS/NotificationCenter/actions/workflows/npm-publish.yml/badge.svg)](https://github.com/DynamoDS/NotificationCenter/actions/workflows/npm-publish.yml)
+
 Dynamo Notification Center WebApp which is leveraged in Dynamo. This can also be leveraged by any products that needs a notification center.
 
 ---
+
 ## Requirements
 
 For development, you will only need Node.js and a node global package, installed in your environement.
 
 ### Node
+
 - #### Node installation on Windows
 
-  Just go on [official Node.js website](https://nodejs.org/) and download the LTS installer.
-Also, be sure to have `git` available in your PATH, `npm` might need it (You can find git [here](https://git-scm.com/)).
+  Just go on [official Node.js website](https://nodejs.org/) and download the installer. Also, be sure to have `git` available in your PATH, `npm` might need it (You can find git [here](https://git-scm.com/)).
```

Comment:
I think I mentioned LTS installer on purpose since we do not have node 18.x Github Action support by default